packages/markers/src/browser/problem/problem-decorator.ts

```diff
@@ -134,6 +134,12 @@ export class ProblemDecorator implements TreeDecorator {
             fontData: {
                 color,
             },
+            tailDecorations: [
+                {
+                    data: 'allo',
```

Comment:
Is this `data` field meaningful?

Reply:
Please remove this extraneous `data` that I assume was for testing purposes.

Reply:
Sorry Colin for not answering your question, I completely forgot. Indeed, it was for testing purposes

Reply:
Me and Vince planned to remove it before merging, we used it only to check if the icon would have the color of the highest priority. Problem markers shouldn't be in my PR since Vince had a commit about them. It should be dropped before merging mine.